src/abstractdataframe/sort.jl

```diff
@@ -223,29 +223,9 @@ ordering(df::AbstractDataFrame, cols::Tuple, args...) = ordering(df, [cols...],
 # Default sorting algorithm
 ###########################
 
-# TimSort is fast for data with structure, but only if the DataFrame is large enough
```

Comment:
Please run at least one reasonable benchmark to check that this doesn't totally kill performance. Also file an issue linking to that code so that we don't forget it.